Makefile

```diff
@@ -98,6 +98,8 @@ bin/$(OS)_$(ARCH)/$(BIN): build-dirs
 	    -v $$(pwd)/bin/$(OS)_$(ARCH):/go/bin/$(OS)_$(ARCH)                       \
 	    -v $$(pwd)/.go/std/$(OS)_$(ARCH):/usr/local/go/pkg/$(OS)_$(ARCH)_static  \
 	    -v $$(pwd)/.go/cache:/.cache                                             \
+            --env HTTP_PROXY=$(HTTP_PROXY)                                 	     \
```

Comment:
indenting is messed up.  Please use tabs to the general indent level, then spaces.  In this case one tab to align with the command (@docker) and then 4 spaces.  Also use spaces as trailing whitespace.

What I see in my editor:


```
>-------@docker run                                                                  \
>-------    -i                                                                       \
>-------    -u $$(id -u):$$(id -g)                                                   \
>-------    -v $$(pwd)/.go:/go                                                       \
>-------    -v $$(pwd):/go/src/$(PKG)                                                \
>-------    -v $$(pwd)/bin/$(OS)_$(ARCH):/go/bin                                     \
>-------    -v $$(pwd)/bin/$(OS)_$(ARCH):/go/bin/$(OS)_$(ARCH)                       \
>-------    -v $$(pwd)/.go/std/$(OS)_$(ARCH):/usr/local/go/pkg/$(OS)_$(ARCH)_static  \
>-------    -v $$(pwd)/.go/cache:/.cache                                             \
            --env HTTP_PROXY=$(HTTP_PROXY)                                 >----     \                                                                                     
            --env HTTPS_PROXY=$(HTTPS_PROXY)                                         \
>-------    -w /go/src/$(PKG)                                                        \
>-------    --rm                                                                     \
>-------    $(BUILD_IMAGE)                                                           \
>-------    /bin/sh -c "                                                             \
>-------        ARCH=$(ARCH)                                                         \
>-------        OS=$(OS)                                                             \
>-------        VERSION=$(VERSION)                                                   \
>-------        PKG=$(PKG)                                                           \
>-------        ./build/build.sh                                                     \
>-------    "
```

Reply:
BTW: `>-------` is a tab :)